app/src/main/java/dev/baseio/discordjetpackcompose/ui/components/DirectMessageList.kt

```diff
@@ -68,7 +69,11 @@ fun DirectMessageList(
             }
         }
         val cardElevation by animateDpAsState(targetValue = if (shouldLiftCard) 2.dp else 0.dp)
-        Card(elevation = cardElevation) {
+        Card(
+            elevation = cardElevation,
+            backgroundColor = DiscordColorProvider.colors.secondaryBackground,
+            contentColor = DiscordColorProvider.colors.contentColorFor(DiscordColorProvider.colors.secondaryBackground)
```

Comment:
@Anmol92verma In the original design (as shown in the bottom image), the `Direct Messages` top section doesn't have a background color. It only has an elevation when the list is scrolled. So should we update it here as well?

<img src="https://i.imgur.com/Q8QZ51E.jpg" width="300"/>